packages/auth/src/features/verifyCode/service.ts

```diff
@@ -74,8 +74,9 @@ export async function sendVerificationCode(
       Authorization: `Bearer ${await createToken(
         'auth',
         ['service'],
-        ['opencrvs:notification-user'],
-        JWT_ISSUER
+        ['opencrvs:notification-user', 'opencrvs:countryconfig-user'],
+        JWT_ISSUER,
+        true
```

Comment:
Adding true means that this JWT is only valid for 10 minutes.  It should have been there before